Authors:  Cecil Curry <leycec@gmail.com>
Upstream: no
Synopsis: Prevent jaring of removed subcomponents.
--- old/build.xml	2010-04-15 10:25:54.784062451 +0000
+++ new/build.xml	2010-04-15 11:09:59.405843314 +0000
@@ -313,36 +313,13 @@
     <jarIt jarfile="${jsp-api.jar}" filesId="files.jsp-api"
       manifest="${tomcat.manifests}/jsp-api.jar.manifest" />
 
-
     <!-- JSP 2.1 EL Implementation JAR File -->
     <jarIt jarfile="${el-api.jar}" filesId="files.el-api"
       manifest="${tomcat.manifests}/el-api.jar.manifest" />
 
-    <!-- Bootstrap JAR File -->
-    <jarIt jarfile="${bootstrap.jar}" filesId="files.bootstrap"
-      manifest="${tomcat.manifests}/bootstrap.jar.manifest" />
-
     <!-- Tomcat-juli JAR File -->
     <jarIt jarfile="${tomcat-juli.jar}" filesId="files.tomcat-juli" />
 
-    <!-- Catalina Main JAR File -->
-    <jarIt jarfile="${catalina.jar}" filesId="files.catalina" />
-
-    <!-- Catalina GroupCom/Tribes JAR File -->
-    <jarIt jarfile="${catalina-tribes.jar}" filesId="files.catalina-tribes" />
-
-    <!-- Catalina Cluster/HA JAR File -->
-    <jarIt jarfile="${catalina-ha.jar}" filesId="files.catalina-ha" />
-
-    <!-- Catalina Ant Tasks JAR File -->
-    <jarIt jarfile="${catalina-ant.jar}" filesId="files.catalina-ant" />
-
-    <!-- Protocol handlers - Coyote -->
-    <jarIt jarfile="${tomcat-coyote.jar}" filesId="files.tomcat-coyote" />
-
-    <!-- Jasper Implementation JAR File -->
-    <jarIt jarfile="${jasper.jar}" filesId="files.jasper" />
-
     <!-- Jasper EL Implementation JAR File -->
     <jarIt jarfile="${jasper-el.jar}" filesId="files.jasper-el" />
 
